internal/inventory/tagprocessor.go

```diff
@@ -52,21 +52,23 @@ func NewTagProcessor(lc logger.LoggingClient, cfg ConsulConfig, tags []StaticTag
 	return tp
 }
 
+func (tp *TagProcessor) IsDebugLogging() bool {
+	return tp.lc.LogLevel() == models.DebugLog || tp.lc.LogLevel() == models.TraceLog
+}
+
 // UpdateConfig takes in a ConsulConfig raw config object and converts it into a locally cached
 // version that is understood by the TagProcessor. It also generates the correct mobility profile
 // based on the supplied values, and the alias map as well.
-func (tp *TagProcessor) UpdateConfig(cfg ConsulConfig) {
-	as := cfg.ApplicationSettings
+func (tp *TagProcessor) UpdateConfig(cfg CustomConfig) {
+	as := cfg.AppSettings
 	profile := newMobilityProfile(as.MobilityProfileSlope, as.MobilityProfileThreshold, as.MobilityProfileHoldoffMillis)
 	aliases := cfg.Aliases
 	delete(aliases, "")
 
-	logLevel := strings.ToUpper(cfg.Writable.LogLevel)
 	tp.config = processorConfig{
 		adjustLastReadOnByOrigin: as.AdjustLastReadOnByOrigin,
 		departedThresholdSeconds: as.DepartedThresholdSeconds,
 		ageOutHours:              as.AgeOutHours,
-		debugLogEnabled:          logLevel == contract.DebugLog || logLevel == contract.TraceLog,
```

Comment:
You need to remove this variable altogether, not just remove the setting of its value. It Also needs to be replaced with the functionality provided via https://github.com/edgexfoundry/go-mod-core-contracts/pull/394/files

Reply:
reworked to use new functionality from go mod contracts